scripts/app.js

```diff
@@ -0,0 +1,304 @@
+const { promisify } = require("util");
+const exec = promisify(require("child_process").exec);
+const fs = require("fs");
+const dotenv = require("dotenv");
+const result = dotenv.config();
```

Comment:
There should be some validation of the config and a clean error message (rather than a stack trace) reporting what's missing in the .env/whether the .env is missing.